### ISSUE TITLE
loading model file error

### ISSUE THREAD

**Issue by kiminh** [xueyouluo/speller-bert#1]
hi, I've download the pretrained model file on baidu yunpan and change the model save file path to D:\nlpmodeldata\cbert\cbert.

python3.7,tensorflow 1.15.0, windows

run 'python serving.py' and get the error

![image](https://user-images.githubusercontent.com/8201408/155478037-546a2593-0968-4c09-9a6b-b2529af5d7f0.png)

> 2022-02-24 15:27:00.092167: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open D:\nlpmodeldata\cbert\cbert: Unknown: NewRandomAccessFile f
ailed to Create/Open: D:\nlpmodeldata\cbert\cbert : 拒绝访问。
; Input/output error
2022-02-24 15:27:00.097491: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open D:\nlpmodeldata\cbert\cbert: Unknown: NewRandomAccessFile f
ailed to Create/Open: D:\nlpmodeldata\cbert\cbert : 拒绝访问。
; Input/output error
2022-02-24 15:27:00.100192: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at save_restore_tensor.cc:175 : Data loss: Unable to open
table file D:\nlpmodeldata\cbert\cbert: Unknown: NewRandomAccessFile failed to Create/Open: D:\nlpmodeldata\cbert\cbert : 拒绝访问。
; Input/output error

**Comment by xueyouluo** [comment]
我看你的解压路径的话，模型路径是D:\nlpmodeldata\cbert\cbert\cbert

**Comment by kiminh** [comment]
fixed. thx a lot~
